dashboard/test/ui/features/star_labs/applab/scenarios.feature

```diff
@@ -49,36 +49,39 @@ Feature: App Lab Scenarios
     And I wait until element "#divApplab > .screen > div#text_area1" is visible
     Then element "div#text_area1" has html "Line 1<div>Line 2</div><div><br></div><div>Line3</div>"
 
-  Scenario: Change event works in text input and text area
+  Scenario: Change event works in text input
     Given I switch to design mode
     And I drag a TEXT_INPUT into the app
-    And I drag a TEXT_AREA into the app
     And I switch to code mode
     And I ensure droplet is in text mode
     And I append text to droplet "onEvent('text_input1', 'change', function(event) {\n"
     And I append text to droplet "  console.log(event.targetId + ': ' + getText('text_input1'));\n"
-    And I append text to droplet "});\n\n"
-    And I append text to droplet "onEvent('text_area1', 'change', function(event) {\n"
-    And I append text to droplet "  console.log(event.targetId + ': ' + getText('text_area1'));\n"
     And I append text to droplet "});"
 
     # in text input, blur produces a change event
     When I press "runButton"
-    And I wait until element "#divApplab > .screen > div#text_area1" is visible
-    And I press keys "123" for element "#text_input1"
+    And I wait until element ".screen > input" is visible
+    And I press keys "123" for element ".screen > input"
     And I blur selector "#text_input1"
     Then element "#debug-output" has escaped text "\"text_input1: 123\""
 
     # in a text input, enter produces a change event but then blur does not
-    When I press keys "456\n" for element "#text_input1"
+    When I press keys "456\n" for element ".screen > input"
     Then element "#debug-output" has escaped text "\"text_input1: 123\"\"text_input1: 123456\""
     And I blur selector "#text_input1"
     Then element "#debug-output" has escaped text "\"text_input1: 123\"\"text_input1: 123456\""
 
+Scenario: Change event works in text area
+    Given I switch to design mode
+    And I drag a TEXT_AREA into the app
+    And I switch to code mode
+    And I ensure droplet is in text mode
+    And I append text to droplet "onEvent('text_area1', 'change', function(event) {\n"
+    And I append text to droplet "  console.log(event.targetId + ': ' + getText('text_area1'));\n"
+    And I append text to droplet "});"
+
     # in a text area, blur produces a change event. sending keystrokes (especially 'enter')
     # to a contentetiable div was too hard to test here due to browser differences.
```

Comment:
What word is `contentetiable` meant to be? contenteditable, maybe? 😅

Reply:
Yes - I think so! I can fix this existing typo.